lib/collector/facts.js

```diff
@@ -30,6 +36,17 @@ function facts(agent, callback) {
       const systemInfo = data.systemInfo || Object.create(null)
       const environment = data.environment || []
 
+      const packages = data.environment.find((elt) => elt[0] === 'Packages')[1]
+      const dedupPackages = new Set()
+      packages.forEach((dep) => dedupPackages.add(JSON.parse(dep)[0]))
+      LOG_LIBRARIES.forEach((lib) => {
+        if (dedupPackages.has(lib)) {
+          agent.metrics
+            .getOrCreateMetric(`${NAMES.SUPPORTABILITY.DEPENDENCIES}/Nodejs/${lib}`)
```

Comment:
we may want a new constant instead of appending `Nodejs` here.  maybe in `lib/metrics/names` we add

`INSTALLED_DEPS`?

Reply:
Sure, let's do that.